src/shared_modules/content_manager/tests/unit/CtiDownloader_test.cpp

```diff
@@ -162,6 +164,65 @@ TEST_F(CtiDownloaderTest, BaseParametersDownloadWithRetry)
     EXPECT_EQ(parameters->lastSnapshotOffset.value(), 3);
 }
 
+/**
+ * @brief Tests the correct download of the parameters with the retry feature when a "too many requests" error is
+ * received.
+ *
+ */
+TEST_F(CtiDownloaderTest, BaseParametersDownloadWithRetryTooManyRequestsError)
+{
+    // Push error.
+    m_spFakeServer->pushError(429);
+
+    auto downloader {CtiDummyDownloader(HTTPRequest::instance(), 1)};
```

Comment:
Nitpick:
We could declare a constexpr for the '1', just to give it a name, and to know at a glance what it means.

Reply:
Done!